packages/babel-plugin-proposal-regexp-modifiers/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@babel/plugin-proposal-regexp-modifiers",
+  "version": "7.19.1",
+  "description": "TODO",
```

Comment:
Help Needed: I'm not sure how I should describe this, thanks!

Reply:
```suggestion
  "description": "Compile inline regular expression modifiers",
```